apps/meteor/tests/data/livechat/department.ts

```diff
@@ -3,105 +3,167 @@ import { expect } from 'chai';
 import type { ILivechatDepartment, IUser, LivechatDepartmentDTO } from '@rocket.chat/core-typings';
 import { api, credentials, methodCall, request } from '../api-data';
 import { IUserCredentialsHeader } from '../user';
-import { createAnOnlineAgent } from './users';
+import { createAnOfflineAgent, createAnOnlineAgent } from './users';
 import { WithRequiredProperty } from './utils';
 
 export const NewDepartmentData = ((): Partial<ILivechatDepartment> => ({
-    enabled: true,
-    name: `new department ${Date.now()}`,
-    description: 'created from api',
-    showOnRegistration: true,
-    email: faker.internet.email(),
-    showOnOfflineForm: true,
+	enabled: true,
+	name: `new department ${Date.now()}`,
+	description: 'created from api',
+	showOnRegistration: true,
+	email: faker.internet.email(),
+	showOnOfflineForm: true,
 }))();
 
 export const createDepartment = async (departmentData: Partial<ILivechatDepartment> = NewDepartmentData): Promise<ILivechatDepartment> => {
-    const response = await request.post(api('livechat/department')).set(credentials).send({
-        department: departmentData,
-    }).expect(200);
-    return response.body.department;
-};
-
-export const updateDepartment = async (departmentId: string, departmentData: Partial<LivechatDepartmentDTO>): Promise<ILivechatDepartment> => {
-    const response = await request.put(api(`livechat/department/${ departmentId }`)).set(credentials).send({
-        department: departmentData,
-    }).expect(200);
-    return response.body.department;
+	const response = await request
+		.post(api('livechat/department'))
+		.set(credentials)
+		.send({
+			department: departmentData,
+		})
+		.expect(200);
```

Comment:
Ik these are style changes, but are they needed? 👀

Reply:
If needed, can you open them on a diff PR? (That one should get quicker reviews)